src/app.js

```diff
@@ -1,4 +1,4 @@
-const cluster = require("cluster");
+// this file never run on the master thread
```

Comment:
```suggestion
// this file never runs on the master thread
```
slight grammar adjustment :)